transport/ws/src/main/java/org/kaazing/gateway/transport/ws/extension/WsExtensionValidation.java

```diff
@@ -23,7 +23,7 @@
 
 public class WsExtensionValidation {
 
```

Comment:
nite: isn't the static implicit/redundant.  See [here](http://stackoverflow.com/questions/23127926/static-enum-vs-non-static-enum)?